### PR TITLE
refactor: sold and hide item

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

File: src/common/minting/enums/item-status.enum.ts [hunk 1]
```diff
@@ -1,10 +1,7 @@
 enum ItemStatus {
   NotListed = 0,
   Listed = 1,
-  Sold = 2,
   Draft = 3,
```
Comment (mmiranda-stackit) [inline src/common/minting/enums/item-status.enum.ts:4]: Make Draft = 2

File: src/components/search-form/flyout-search-form.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const SearchForm = ({ isOpen }: Props) => (
       <button type="submit" className="search-button">
         <i className="feather-search" />
       </button>
-      <input type="text" placeholder="Search ..." />
+      <input type="text" placeholder="Search ...xx" />
```
Comment (mmiranda-stackit) [inline src/components/search-form/flyout-search-form.tsx:18]: What does this mean? xx

File: src/containers/author-profile/index.tsx [hunk 3]
```diff
@@ -27,10 +26,7 @@ const AuthorProfileArea = ({ className, address }: Props) => {
   const ownedItems = useAppSelector((state) => selectOwnedItems(state, address));
 
   const likedItemsToShow = useMemo(
-    () =>
-      likedItems.filter(
-        (item) => item.status !== ItemStatus.Hidden || item.owner.address === address
-      ),
+    () => likedItems.filter((item) => item.isHidden || item.owner.address === address),
```
Comment (mmiranda-stackit) [inline src/containers/author-profile/index.tsx:29]: This condition should be the other way around. Have a look at how it used to be

File: src/features/marketplace/store/marketplace.slice.ts [hunk 4]
```diff
@@ -227,10 +227,7 @@ export const selectCanIList = (state: RootState) => {
     auth: { address },
     marketplace: { selectedItem },
   } = state;
-  return (
-    selectedItem.owner.address === address &&
-    [ItemStatus.NotListed, ItemStatus.Sold, ItemStatus.Visible].includes(selectedItem.status)
-  );
+  return selectedItem.owner.address === address && selectedItem.status === ItemStatus.NotListed;
```
Comment (mmiranda-stackit) [inline src/features/marketplace/store/marketplace.slice.ts:230]: You should include the isHidden = false here

File: src/features/marketplace/store/marketplace.slice.ts [hunk 5]
```diff
@@ -248,9 +245,7 @@ export const selectCanISeeItem = (state: RootState) => {
 
   const isOwner = selectedItem?.owner?.address === address;
 
-  const isAbleTosee = [ItemStatus.Listed, ItemStatus.NotListed, ItemStatus.Visible].includes(
-    selectedItem?.status
-  );
+  const isAbleTosee = [ItemStatus.Listed, ItemStatus.NotListed].includes(selectedItem?.status);
```
Comment (mmiranda-stackit) [inline src/features/marketplace/store/marketplace.slice.ts:248]: Include the isHidden = false

**Review by mmiranda-stackit — CHANGES_REQUESTED** [review]

File: src/components/widgets/social-widget/index.tsx [hunk 6]
```diff
@@ -1,5 +1,4 @@
 import { SocialItem } from '@types';
-import * as FontAwesomeIcons from 'react-icons/fa';
 
 const SocialWidget = ({ id, icon, link, title }: SocialItem) => (
   /* const Icon = FontAwesomeIcons[icon]; */
```
Comment (juliswer) [inline src/components/widgets/social-widget/index.tsx:4]: remove this comment

**Review by juliswer — DISMISSED** [review]

**Review by mmiranda-stackit — APPROVED** [review]

**Pull request merged**